src/pages/Login/Login.tsx

```diff
@@ -13,35 +17,60 @@ type LoginProps = {
   setUserId: (userId: string) => void;
 };
 
+
 const Login = ({
   email,
   setEmail,
   password,
   setPassword,
-  setUserId,
+  setUserId
 }: LoginProps) => {
-  const navigate = useNavigate();
+  const navigate = useNavigate()
+  const [loginError, setLoginError] = useState<boolean>(false);
 
   const handleLogin = async () => {
     try {
       const userData = await signInWithEmailAndPassword(auth, email, password);
       setUserId(userData.user.uid);
-      navigate("/home");
+      navigate("/home")
```

Comment:
I do not think you want to navigate to "/home" here as that is not the route for `Home` anymore